ansible/group_vars/all.yml.sample

```diff
@@ -1,9 +1,9 @@
 ---
 # Values accepted: 'latest-4.5', 'latest-4.6', explicit version i.e. 4.3.0-0.nightly-2019-12-09-035405 (must be defined)
-version: 4.6.1
```

Comment:
We don't want to change the default sample file, assuming this got committed inadvertently.